thrust/cmake/ThrustUtilities.cmake

```diff
@@ -31,13 +31,8 @@ function(thrust_configure_cuda_target target_name)
   endif()
 
   if (thrust_cuda_RDC)
-    set_target_properties(${target_name} PROPERTIES
-      CUDA_ARCHITECTURES "${THRUST_CUDA_ARCHITECTURES_RDC}"
-      POSITION_INDEPENDENT_CODE ON
-      CUDA_SEPARABLE_COMPILATION ON)
+    set_target_properties(${target_name} PROPERTIES CUDA_SEPARABLE_COMPILATION ON POSITION_INDEPENDENT_CODE ON)
```

Comment:
nitpick: Leave the separate properties on separate lines for better readability.